build.sbt

```diff
@@ -41,11 +27,29 @@ lazy val utilRoot: Project = (project in file(".")).
     utilLogging, utilRelation, utilLogic, utilCache, utilTracking
   ).
   settings(
-    buildLevelSettings,
+    inThisBuild(Seq(
+      organization := "org.scala-sbt.util",
```

Comment:
I probably should have mentioned this earlier, but I'm never a fan of calling something "util".  This is akin to "misc" where just general cruft acrues in the package, or you never know if something does or does not belong in it.

In any case, just a minor rant/surprise at seeing the "util" in the organization.